t/gcp_project_services/README.md

```diff
@@ -0,0 +1,19 @@
+# Example: GCP Project Services
+
+Enables GCP services for a specific project in an environment specified by the [gcp-org-terraform-template](https://github.com/abcxyz/gcp-org-terraform-template).
+
+1. [Install the `abc` binary](https://github.com/abcxyz/abc/blob/main/README.md#installation).
+
+1. cd into the resources/[product_id] directory
+
+    ```shell
+    $ cd gcp-org-terraform-template/resources/[product_id]
```

Comment:
I would rather see this specified in the dest flag, this allows the user flexibility in the destination because their folder probably wont be called `gcp-org-terraform-template`.